resources/dependencies.rb

```diff
@@ -17,7 +17,5 @@
 #
 
 action :install do
-  openjdk_install '8' do
-    action :install
-  end
+  openjdk_install '8'
```

Comment:
I think you missed the other change I asked which was using the `adoptopenjdk_install` instead.
```suggestion
  adoptopenjdk_install '8'
```

Reply:
oops sorry about that. Read to fast. Fixed in latest commit.